v56-team22-surgery-status-board/src/constant/nav.ts

```diff
@@ -0,0 +1,21 @@
+
+export type Role = 'guest' | 'surgical' | 'admin';
+
+interface Navlink {
+  label: string;
+  href: string;
+}
+
+export const navByRole: Record<Role, Navlink[]> = {
+  guest: [{ label: 'Patient Status', href: '/patient-status' }],
```

Comment:
so this might be work for another ticket because we haven't definitively defined routes. but I think `update-patient` could be confusing between the patient status and the patient information. I'm willing to leave it for now and wait to update the router.